code/ui/src/common/websocket.js

```diff
@@ -0,0 +1,80 @@
+// #if process.env.NODE_ENV === 'development'
```

Comment:
Q: which part of the build process understands this line?
Looking at babel API though current UI, I can use `if (process.env.BLA === "false")` and eliminate that branch with babel-minify (and I think transform-inline-environment-variables is also required?)

Reply:
A: https://www.npmjs.com/package/webpack-conditional-loader
(figures)

Reply:
Yep that's the one